content/en/docs/concepts/configuration/secret.md

```diff
@@ -314,18 +301,20 @@ The output is similar to:
 1f2d1e2e67df
 ```
 
-The program in a container is responsible for reading the secrets from the
-files.
+The program in a container is responsible for reading the secret data from these
+files, as needed.
 
-#### Mounted Secrets are updated automatically
+##### Mounted Secrets are updated automatically
 
-When a secret currently consumed in a volume is updated, projected keys are eventually updated as well.
-The kubelet checks whether the mounted secret is fresh on every periodic sync.
+When a Secret currently consumed in a volume is updated, projected keys are eventually
+updated as well.  
```

Comment:
nit: Do you need the word, "currently"?
What do you call a "Secret consumed in a volume"?

Reply:
Maybe “When you present data from a Secret to a container using a `secret` volume, &hellip;” ?

Reply:
Possibly. The same sentence with minor tweaks:
When a Secret stored in a volume is updated, the projected keys are eventually updated as well.